core/core.go

```diff
@@ -97,17 +97,17 @@ func (c *Core) Stop() {
 // Slice methods //
 //---------------//
 
-func (c *Core) Append(header *types.Header, domTerminus common.Hash, td *big.Int, domOrigin bool, reorg bool) (types.PendingHeader, error) {
-	return c.sl.Append(header, domTerminus, td, domOrigin, reorg)
+func (c *Core) Append(header *types.Header, pendingHeader *types.Header, domTerminus common.Hash, td *big.Int, domOrigin bool, reorg bool) error {
```

Comment:
would prefer args passed down from dom to sub, to be named as such. Maybe `domPendingHeader`? Just how `domTerminus` is the terminus passed in from dom?

Not a show stopper, but it would be a nice touch for reading the code in all of these functions (through core, slice, hc, etc)

Reply:
I agree, it is better for code readability. Changing it.

Reply:
Changed this in the API, client and backend as well. THanks